test/output.jl

```diff
@@ -0,0 +1,139 @@
+@testset "output" begin
+    if Metal.macos_version() >= v"15"
+    @testset "formatted output" begin
```

Comment:
I would add a warning when skipping the tests and reduce indentation by one level for consistency with other tests.

The reduced indentation suggestions applies to the whole file.

```suggestion
@testset "output" begin

if Metal.macos_version() < v"15"

@warn "Skipping output tests in macOS 14 and below"

else

@testset "formatted output" begin
```